tests/providers/google/marketing_platform/operators/test_search_ads_system.py

```diff
@@ -17,8 +17,8 @@
 # under the License.
```

Comment:
In the new design this file is not required. Please ensure that the setup/teardown fixtures are migrated to the DAG itself.